CONTRIBUTING.md

```diff
@@ -55,143 +69,61 @@ To set up an environment to start developing on conda code, we recommend the fol
-Contribution, or the Work to which You have contributed, constitutes direct or contributory patent
-infringement, then any patent licenses granted to that entity under this Agreement for that
-Contribution or Work shall terminate as of the date such litigation is filed.
-
-4. You represent that you are legally entitled to grant the above license. If your employer(s) has
-rights to intellectual property that you create that includes your Contributions, You represent
-that you have received permission to make Contributions on behalf of that employer, that your
-employer has waived such rights for your Contributions to Anaconda, or that your employer has
-executed a separate Corporate Contributor License Agreement with Anaconda. 
-
-5. You represent that each of Your Contributions is Your original creation (see Section 7 for
-submissions on behalf of others). You represent that Your Contribution submissions include
-complete details of any third-party license or other restriction (including, but not limited to,
-related patents and trademarks) of which you are personally aware and which are associated with
-any part of Your Contributions. 
-
-6.	You are not expected to provide support for Your Contributions, except to the extent You
-desire to provide support. You may provide support for free, for a fee, or not at all. Unless
-required by applicable law or agreed to in writing, You provide Your Contributions on an "AS IS"
-BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied, including, without
-limitation, any warranties or conditions of TITLE, NONINFRINGEMENT, MERCHANTABILITY, or FITNESS
-FOR A PARTICULAR PURPOSE. 
-
-7. Should You wish to submit work that is not Your original creation, You may submit it to
-Anaconda separately from any Contribution, identifying the complete details of its source and of
-any license or other restriction (including, but not limited to, related patents, trademarks,
-and license agreements) of which you are personally aware, and conspicuously marking the work as
-"Submitted on behalf of a third-party: [named here]".
-
-8. You agree to notify Anaconda of any facts or circumstances of which you become aware that
-would make these representations inaccurate in any respect.
+In case you're new to CLAs, this is rather standard procedure for larger
+projects. [Django](https://www.djangoproject.com/foundation/cla/) and
+[Python](https://www.python.org/psf/contrib/contrib-form/) for example
+both use similar agreements.
+
+Note: New contributors are required to complete the [Conda Contributor License Agreement][1].
+
+For pull requests to be merged, contributors to GitHub pull requests need to
+have signed the [Conda Contributor License Agreement][1], so Anaconda, Inc.
+has it on file. A record of prior signatories is kept in a [separate repo in
+conda's GitHub][2] organization.
+
+[1]: https://conda.io/en/latest/contributing.html#conda-contributor-license-agreement
```

Comment:
Thanks for removing the duplicated and yet out of date CLA text!

Reply:
No problem!